model/Core/Classes/Dictionary.md

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DictionaryEntry
+
+## Summary
+
+A key with an associated value.
+
+## Description
+
+The class used for implementing a generic string mapping (also known as associative arrays, dictionaries, and hash maps) in SPDX.  To implement a dictionary, this class would be used in a collection with unique keys. Each DictionaryEntry would contain a key-value pair which maps the key to its associated value.  Both the key and the value are strings.
```

Comment:
```suggestion
The class used for implementing a generic string mapping (also known as associative arrays, dictionaries, and hash maps) in SPDX.  Each DictionaryEntry contains a key-value pair which maps the key to its associated value.  Both the key and the value are strings.  To implement a dictionary, this class is to be used in a collection with unique keys.
```
I don't think the subjunctive makes sense here.
I also suggest the above sentence reordering: In the current version, the particular concept of `key` in the DictionaryEntry is only first introduced in the sentence after it is referred to.